DOCS-2847 RUM Unified Explorer and Saved Views Documentation (#12901)
DataDog/documentation: content/en/real_user_monitoring/explorer/visualize.md

---
title: Visualize
kind: documentation
further_reading:
- link: "/real_user_monitoring/explorer/search/"
  tag: "Documentation"
  text: "Search for your events"
---

## Overview

Visualizations define the outcomes of the filters and aggregates displayed in the [RUM Explorer][1]. Select the relevant visualization type for you to surface the information you need under the search query.

## Lists

Lists are paginated results of events and are ideal when individual results matter. You do not need prior knowledge of what defines a matching result to use lists.

{{< img src="real_user_monitoring/explorer/visualize/rum_explorer_lists.mp4" alt="Lists in the RUM Explorer" video="true" style="width:70%;" >}}

> The information you search for is displayed in columns. You can manage these table columns with:

Author: ```suggestion
The information you search for is displayed in columns. You can manage the following:
```

- The table with available interactions in the first row. You can sort, rearrange, and remove columns.
- The facet panel on the left or the RUM [event side panel][2] on the right. You can add a column for a field. 

By default, RUM events in the list visualization are organized by timestamp with the most recent events listed first. You can sort events in any way you want, such as with facets. Surface RUM events with the lowest or highest value for a measure first, then sort your events lexicographically for the unique value of a facet. This orders a column according to the facet.

> While you can add attributes and tags in columns, Datadog recommends sorting the table if you have [declared a facet][3] first. You can add non-faceted attributes in the columns but it may not sort correctly.

Author: @amina196 do you have any input on this edge case?

> While you can add attributes and tags in columns, Datadog recommends sorting the table if you have [declared a facet][3] first. You can add non-faceted attributes in the columns but it may not sort correctly.

Reviewer: because you might need to see the value for a given custom attribute for a line item in the table. So it's not an edge case. We just are not making guarantees that the sorting will work well if no facet exists (facets help us get an understanding of what the field contains, whether it's a number of a string, and have the right behavior). We don't block it because we might get it right sometimes and want to push people to create facets so that they can get the most value of the Explorer. Does that make sense?
